dvm.ps1

```diff
@@ -5,7 +5,7 @@
 function dvm() {
   $dvmDir = $PSScriptRoot
 
-  if( !(Test-Path (Join-Path $dvmDir dvm-helper.exe)) ) {
+  if( !(Test-Path (Join-Path $dvmDir dvm-helper dvm-helper.exe)) ) {

```

Comment:
`Join-Path` is not as awesome as it should be and doesn't accept more than 2 arguments... (grumbles about powershell). This will work though:

```powershell
Join-Path $dvmDir dvm-helper\dvm-helper.exe
```